easybuild/easyconfigs/s/spaceranger/spaceranger-1.1.0-GCC-9.3.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
+
+name = 'spaceranger'
+version = '1.1.0'
+
+homepage = 'https://support.10xgenomics.com/spatial-gene-expression/software/downloads/latest'
```

Based on PR #11748, we could also use here the `What is` page as homepage
```suggestion
homepage = 'https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger'
```